.github/workflows/build.yml

```diff
@@ -302,11 +302,22 @@ jobs:
           echo CONFIG_TARGET_PER_DEVICE_ROOTFS=y >> .config
           echo CONFIG_TARGET_ALL_PROFILES=y >> .config
 
-      - name: Configure ccache
+        # ccache for some reason have problem detecting compiler type
```

Comment:
Must have been fun to debug this... did you inform upstream? If so maybe add a link to the mailing list/ticket so we can eventually drop this workaround again :)

Reply:
Ehh IMHO not worth it. From what I notice ccache use the compile name to match and try to find the type... And with external toolchain we add a special name so the matching fails. So we need to force it. All documented in the commit message